app/models/crop.rb

```diff
@@ -47,8 +47,7 @@ class Crop < ActiveRecord::Base
     format: {
       with: %r{\Ahttps?:\/\/en\.wikipedia\.org\/wiki\/[[:alnum:]%_\.()-]+\z},
       message: 'is not a valid English Wikipedia URL'
-    },
-    if: :approved?
+    }, presence: true
```

Comment:
the form wasn't enforcing that you need a wikipedia url.. I remember it used to.
the markup on the form says it's required, so let's make it required.